resources/canary-release.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable */
```

Comment:
This is optional, we are using that to do custom pre-release and not use the default way of doing that with `changesets`...